src/averageAges.js

```diff
@@ -14,14 +14,23 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
-}
+const MALE = 'f';
```

Comment:
The constant `MALE` is incorrectly assigned the value 'f', which stands for female. It should be 'm' to correctly represent male.